### PR TITLE
Round floats to the nearest integer instead in survey submission.

### PR DESCRIPTION
**Pull request by maurizi**
This seems like more correct behaviour that will lead to better data
quality.

Connects to #1695

### CODE REVIEW

**Comment by kdeloach** [comment]
+1

File: src/nyc_trees/apps/survey/views.py [hunk 1]
```diff
@@ -614,7 +614,7 @@ def _create_survey_and_trees(request, event=None):
         # Convert any floats into integers for all the integer fields in Tree
         for field in ('circumference', 'stump_diameter'):
             if field in tree_data:
-                tree_data[field] = int(float(tree_data[field]))
+                tree_data[field] = int(round(float(tree_data[field])))
```
Comment (hectcastro) [inline src/nyc_trees/apps/survey/views.py]: I haven't been able to find any evidence that `float` adds anything here:

```python
Python 2.7.6 (default, Mar 22 2014, 22:59:56)
[GCC 4.8.2] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> x = 2.33
>>> y = 2.93
>>> int(round(x))
2
>>> int(round(y))
3
>>> int(round(float(x)))
2
>>> int(round(float(y)))
3
```

Comment (maurizi) [inline src/nyc_trees/apps/survey/views.py]: @hectcastro The data is coming in from the client as a string, not a float. (`"2.5"`, not `2.5`).
`float(...)` converts the string to a number.

Comment (hectcastro) [inline src/nyc_trees/apps/survey/views.py]: Good point.

**Comment by hectcastro** [comment]
:+1:

**Pull request merged**